app/assets/javascripts/components/NewsletterSticky.js

```diff
@@ -53,6 +53,12 @@ define([
     this.closeButton = this.$el.find(this.config.closeClassSelector);
     $window = $(window);
 
+    var focusable = this.$el.find('a[href], area[href], input:not([disabled], [type=hidden]), ' +
+      'select:not([disabled], [hidden]), textarea:not([disabled], [hidden]), ' +
+      'button:not([disabled], [hidden]), iframe, object, embed, *[tabindex], *[contenteditable]');
+    this.firstFocusElement = focusable.first();
+    this.lastFocusElement = focusable.last();
```

Comment:
Strictly speaking these are private properties so an underscore prefix helps indicate that. ```this._firstFocusElement```